source/FreeRTOS_IP_Utils.c

```diff
@@ -1050,7 +1050,7 @@ uint16_t usGenerateChecksum( uint16_t usSum,
         else if( ( uxMinLastSize * ipMONITOR_PERCENTAGE_90 ) > ( uxMinSize * ipMONITOR_PERCENTAGE_100 ) )
         {
             uxMinLastSize = uxMinSize;
-            FreeRTOS_printf( ( "Heap: current %lu lowest %lu\n", xPortGetFreeHeapSize(), uxMinSize ) );
+            FreeRTOS_printf( ( "Heap: current %lu lowest %lu\n", (unsigned long)xPortGetFreeHeapSize(), (unsigned long)uxMinSize ) );
```

Comment:
Here I would write:
~~~diff
-FreeRTOS_printf( ( "Heap: current %u lowest %u\n",
-                   xPortGetFreeHeapSize(),
-                   uxMinSize ) );
+FreeRTOS_printf( ( "Heap: current %u lowest %u\n",
+                  ( unsigned ) xPortGetFreeHeapSize(),
+                  ( unsigned ) uxMinSize ) );
~~~
because it will compile well on all platforms.